jmbase/test/test_commands.py

```diff
@@ -21,12 +21,12 @@ def checkClientResponse(self, response):
         is considered criticial.
         """
         if 'accepted' not in response or not response['accepted']:
-            reactor.stop()
+            raise Exception(response)
```

Comment:
after rebasing there was an error in `test_commands.py` which resulted in an extremely unhelpful test fail, with something inside twisted's test running complaining about a `KeyboardInterrupt`. After removing the `reactor.stop()` lines I got useful error messages. Maybe this was needed in some older version but is not any more?

Reply:
I think so (although not 100% sure). Seems to work anyway.